kotlin-emotion/src/main/kotlin/emotion/css/css.kt

```diff
@@ -3,6 +3,6 @@
 
 package emotion.css
 
-import csstype.Properties
+import csstype.PropertiesBuilder
 
-external fun css(properties: Properties): String
+external fun css(properties: PropertiesBuilder): String
```

Comment:
Invalid contract

Reply:
why?

Reply:
https://emotion.sh/docs/@emotion/css#css
```js
  css({
      backgroundColor: 'hotpink',
      '&:hover': { // <- selector
        color
      }
    }
 ```

Reply:
Because `Properties` - also valid

Reply:
Yea, you are right, fixed